front-end/src/components/Navbar.vue

```diff
@@ -19,11 +19,11 @@ function showMenu() {
             </div> 
             <div class="nav-content" :class="this.showMobileMenu ? 'open-menu' : 'closed-menu'"> 
             <div class="logo">
-                <router-link to="/">LOGO</router-link>
+                <router-link to="/">Home</router-link>
             </div> 
                 <ul class="nav-items">
                     <li>
-                        <router-link to="/multiple-choice">Multiple Choice Quiz</router-link>
```

Comment:
I think there needs to be a link to the quiz either on the navbar or the homepage. How else will a user get to the quiz?

 Could you put this link back or add a new one on the homepage?

Reply:
After logging in it will redirect to the quiz and i removed from the navbar because i think its not make any sense to put it on homepage because it its on homepage user can directly go to the quiz without logging in thats why i removed it.